python/lsst/ts/ess/sensors/command_handler.py

```diff
@@ -193,6 +193,21 @@ async def stop_sending_telemetry(self):
             self._ess_instruments.remove(ess_instrument)
         return ResponseCode.OK
 
+    async def _process_sensor_telemetry(self, telemetry):
+        """Convenience coroutine to wrap the telemetry in a dictionary and to
+        pass it on to the callback coroutine. It is up to the callback
+        coroutine to handle the telemetry further (e.g. in case of the
+        SocketServer the telemetry gets sent to the client).
```

Comment:
Nice and clear. I gently suggest reformatting as one brief description, a blank line, then the rest. Also please use active voice. For example:
```
"""Wrap the telemetry in a dict and call the callback coroutine with the result.

The dict has the form `{"telemetry": telemetry}`.
The callback coroutine typically writes its argument to the client.
```
though that begs the question of "how" does it write the data? A dict is not serialized. I suggest you explain *that* in the main class doc string -- where you describe the callback argument. I am guessing you json-encode the data, but that's just a guess.

Reply:
The whole reason for having separate classes for the SocketServer itself and handling of the data is separation of concern. In principles the CommandHandler doesn't have to care about how the data are processed by the callback function. So, I split up the doc string in two parts as you suggested and expanded the doc string of the write method in `SocketServer` a bit.